test/unit/math/rev/functor/ode_rk45_rev_test.cpp

```diff
@@ -265,6 +265,86 @@ TEST(StanMathOde_ode_rk45, scalar_std_vector_args) {
   EXPECT_NEAR(a1[0].adj(), -0.50107310888, 1e-5);
 }
 
+TEST(StanMathOde_ode_rk45, closure_var) {
+  using stan::math::var;
+
+  Eigen::VectorXd y0 = Eigen::VectorXd::Zero(1);
+  double t0 = 0.0;
+  std::vector<double> ts = {1.1};
+
+  var a0 = 0.75;
+  std::vector<var> a1 = {0.75};
+
+  auto f = stan::math::from_lambda(
+      [&](const auto& a, const auto& t, const auto& y, const auto& b,
```

Comment:
Should this have no capture? If it captures by reference it might absorb something we don't want.

Reply:
It's me not being very familiar with C++. I had assumed the capture default was just part of lambda syntax.